src/test/java/core/basesyntax/service/impl/ParseServiceImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ParseServiceImplTest {
+    private ParseServiceImpl parseService;
+
+    @BeforeEach
+    void setUp() {
+        parseService = new ParseServiceImpl();
+    }
+
+    @Test
+    void parseFromString_emptyList() {
```

Comment:
please adhere to the established naming convention
add _ok or _throwsSomeException or just _notOk, third part of method name should be expected result, it will boost readability